### PR TITLE
Migration configuration update

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

* Provide flyway with DB settings.
* Change flyway configuration to start migration from version 001. 

Flyway doesn't see fresh postgres database on Azure as empty (maybe because of some initial functions or views that come with it). For that reason it requires us to set baselineOnMigration to true. That, however, causes another problem, which is the default baseline version (1), which comes after our initial version - 001. This is why the baseline version is set to 000.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -60,5 +60,8 @@ module "send-letter-producer-service" {
     LETTER_TRACKING_DB_USER_NAME  = "${module.db.user_name}"
     LETTER_TRACKING_DB_PASSWORD   = "${module.db.postgresql_password}"
     LETTER_TRACKING_DB_NAME       = "${module.db.postgresql_database}"
+    FLYWAY_URL                    = "jdbc:postgresql://${module.db.host_name}:${module.db.postgresql_listen_port}/${module.db.postgresql_database}"
```
Comment (lgonczar) [inline infrastructure/main.tf]: I could build flyway properties based on the LETTER_TRACKING... ones, but I imagine that at some point they will become different, as the API shouldn't be allowed to change schema.

**Comment by doncem** [comment]
as many amounts as i ran the migration while developing it never ran from 2 migration. are you sure about this?

**Comment by lgonczar** [comment]
Yes, I had this problem whenever I tried to run migration on an empty Azure (postgres) database

**Comment by doncem** [comment]
strange 🤔 . must be azure thing

**Pull request merged**